quotefault/templates/bootstrap/admin.html

```diff
@@ -32,14 +32,14 @@
                 <div class="card-body">
                     "{{ report.quote.quote }}" <b>- {{ get_display_name(report.quote.speaker) }}</b>
                     <br>
-                    <b>Reported by: {{report.reporter}}</b>
+                    <b>Reported by: {{report.reporter}} | Reason: {{report.reason}}</b>
                 </div>
                 <div class="card-footer">
                     Submitted by <a href="https://profiles.csh.rit.edu/user/{{ report.quote.submitter }}">{{ get_display_name(report.quote.submitter) }}</a> on {{ report.quote.quote_time.strftime('%Y-%m-%d %H:%M:%S') }}
-                    <form method="POST" action="/review/{{report.id}}/0" id="hide::{{report.id}}" method="POST">
+                    <form method="POST" action="/review/{{report.id}}/hide" id="hide::{{report.id}}" method="POST">
                         <button type="submit" class="btn btn-danger float-right">Hide</button>
                     </form>
-                    <form method="POST" action="/review/{{report.id}}/1" id="keep::{{report.id}}" method="POST">
+                    <form method="POST" action="/review/{{report.id}}/keep" id="keep::{{report.id}}" method="POST">
                         <button type="submit" class="btn btn-success float-right">Keep</button>
                     </form>
```

Comment:
Hmm. I wonder if buttons and calling out to js would be better than forms. It would also allow for better error handling and we could avoid reloading the page to update results. I'm undecided whether this is something I want to push for in this PR or something we can just push to an issue, thoughts? cc @Mstrodl @WillNilges

Reply:
All issues fixed except for this, imo it seems to good for now but up to you all

Reply:
I do like the idea of avoiding a reload. Lord knows we have enough of that in CSH services. How much work would it be?

Reply:
A little JS, maybe a tiny change to the html template too.

Reply:
Personally, I'm cool with this being a roadmap feature. We have a solid first implementation and people are eager to get quotefault back :upside_down_face:

Reply:
This shouldn't be super hard, I think you should go for it. If you want help with this let me know...